zwave-classifier.js

```diff
@@ -688,35 +711,29 @@ class ZWaveClassifier {
       '',
       'parseAlarmTamperZwValue'
     );
-  }
-
-  addCarbonMonoxideProperty(node, sensorAlarmValueId) {
     this.addProperty(
```

Comment:
It looks like this is still calling addProperty for CO and overheat from inside addAlarmProperty.

addAlarmProperty will only be called if there is a zwValue for command class 113 with an index of 10 (i.e. the key would be 3-113-1-10). This particular key is only used for motion sensors.

So you should remove these additions to addAlarmProperty (and I need to check that addAlarmProperty is even needed at all since I think that everything should be already covered by initSensorNotification.

Reply:
ok, got it :)